airbyte-ci/connectors/pipelines/pipelines/airbyte_ci/connectors/publish/context.py

```diff
@@ -46,12 +46,16 @@ def __init__(
         s3_build_cache_access_key_id: Optional[str] = None,
         s3_build_cache_secret_key: Optional[str] = None,
         use_local_cdk: bool = False,
+        python_registry_token: Optional[str] = None,
+        python_registry_url: Optional[str] = None,
```

Comment:
Can we add a validation on init that these keyword arguments are not None when `self.connector.language in [ConnectorLanguage.Python, ConnectorLanguage.LowCode]`?

Reply:
added a check to the step - if the python publish logic is hit, it will fail with a descriptive error in case the registry or token isn't set properly.